responses.go

```diff
@@ -222,9 +239,18 @@ type AccountResp struct {
 	CPULimit               AccountResourceLimit `json:"cpu_limit"`
 	Permissions            []Permission         `json:"permissions"`
 	TotalResources         TotalResources       `json:"total_resources"`
-	SelfDelegatedBandwidth DelegatedBandwidth   `json:"self_delegated_bandwidth"`
+	SelfDelegatedBandwidth *DelegatedBandwidth  `json:"self_delegated_bandwidth"`
 	RefundRequest          *RefundRequest       `json:"refund_request"`
 	VoterInfo              VoterInfo            `json:"voter_info"`
+
+	// added 2022-11-02
+	HeadBlockNum           uint64               `json:"head_block_num"`
+	HeadBlockTime          BlockTimestamp       `json:"head_block_time"`
+	RexInfo                *RexInfo             `json:"rex_info,omitempty"`
+	SubjectiveSpuBillLimit AccountResourceLimit `json:"subjective_cpu_bill_limit"`
+
+	// TODO: need to set the exact type
```

Comment:
Can I help somehow with this TODO?

Reply:
It would be thanks :)

Reply:
Checked & updated from Leap repo
https://github.com/eoscanada/eos-go/pull/191/commits/9794c6ca7371c08add27485713eab9ad3fb6de3a